### DISCUSSION TITLE
开启对话后，页面滚动较多会遮挡住修改模型下拉框，必须滚到页面最上方才可以再修改模型

### DISCUSSION THREAD

**Discussion by loli0123456789** [open-webui/open-webui#1356] [General]
# Bug Report

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by zishengwu** [comment]
i saw the issue, it maybe helpful for you:
https://github.com/open-webui/open-webui/pull/1352

**Comment by tjbck** [comment]
@loli0123456789 @zishengwu Check out our dev branch, should address your concerns!
